checkClock/config/database.yml

```diff
@@ -0,0 +1,86 @@
+# PostgreSQL. Versions 9.3 and up are supported.
```

Comment:
This file shouldn't be uploaded to GitHub, instead you should put a file with the name database.example.yml, only with the suggested configuration.

Reply:
Ok added `database.yml` to gitignore and created database.example,yml file